<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-P0T0HMV6" failures="1" tests="2" name="_TestNG4_AssertMethods.AssertNull_AssertNotNull" time="7.226" errors="0" timestamp="2022-06-15T11:50:11 IST" skipped="0">
  <testcase classname="_TestNG4_AssertMethods.AssertNull_AssertNotNull" name="CheckUN" time="3.593">
    <failure message="UN Field is not Null TC is Failed. expected [null] but found [[[ChromeDriver: chrome on WINDOWS (60db8ec76b2d95e4ab6ca9949ccdda0e)] -&gt; id: email]]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: UN Field is not Null TC is Failed. expected [null] but found [[[ChromeDriver: chrome on WINDOWS (60db8ec76b2d95e4ab6ca9949ccdda0e)] -> id: email]]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotSame(Assert.java:1033)
at org.testng.Assert.assertNull(Assert.java:965)
at _TestNG4_AssertMethods.AssertNull_AssertNotNull.CheckUN(AssertNull_AssertNotNull.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CheckUN -->
  <system-out/>
  <testcase classname="_TestNG4_AssertMethods.AssertNull_AssertNotNull" name="CheckPass" time="3.633"/>
  <system-out>
    <![CDATA[Password Field is not null TC is Passed.]]>
  </system-out>
</testsuite> <!-- _TestNG4_AssertMethods.AssertNull_AssertNotNull -->
